src/partials/header.html

```diff
@@ -1,20 +1,31 @@
-<div class="header-container">
     <div class="header">
```

Comment:
Marcin, pozbyłeś się zupełnie klasy container. Zostaw go jak poprzednio w oddzielnym div (wtedy prostsze stylowanie oddzielnego diva), albo dodaj klasę do diva z klasą header, potem się zobaczy, czy można będzie je poprawnie wystylizować dla dwóch klas jednocześnie (czy nie będzie jakiegoś nadpisywania) - to rozwiązanie w mojej opinii będzie mniej odpowiednie.

Reply:
Już chyba wiem o co Ci chodzi. :) Zmienione.